.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       with:
         go_version: ${{ inputs.go-version }}
         golangci_lint_version: ${{ steps.linter-version.outputs.version }}
-        fail_level: error
+        fail_level: any
```

Comment:
This replicates the current behavior: https://github.com/reviewdog/reviewdog/blob/master/cmd/reviewdog/main.go#L1045-L1058